doc/sphinx/language/gallina-specification-language.rst

```diff
@@ -144,61 +144,101 @@ presentation of Cic is given in Chapter :ref:`calculusofinductiveconstructions`.
-                    : _
-                    : `num`
-                    : ( `pattern` | … | `pattern` )
+   term : forall `open_binders` , `term`
+        : fun `open_binders` => `term`
+        : `term_let`
+        : if `term` `return_type_opt` then `term` else `term`
+        : `single_fix`
+        : `single_fix` with `fix_body_list` for `ident`
+        : `term100`
+   type_cstr : : `term`
+             : `empty`
+   name : _
+        : `ident`
+   return_type_opt : `as_name_opt` return `term100`
+                   : `empty`
+   as_name_opt : as `name`
+               : `empty`
+   binders_fixannot : { `ident` `impl_ident` } `binders_fixannot`
+                    : `fixannot`
+                    : `binder` `binders_fixannot`
+                    : `empty`
+   impl_ident : `empty`
+              : `name_list` : `term`
+              : `name_list`
+              : : `term`
+   fixannot : { struct `ident` }
+            : { wf `constr` `ident` }
+            : { measure `constr` `ident_opt` `constr_opt` }
+            : { struct `name` }
+            : `empty`
+   name_list : `name_list` `name`
+             : `name`
+   term100 : `term_cast`
+           : `term10` :>
+           : `term10`
+   term10 : `term9` `appl_arg_list`
+          : @ `qualid` `universe_annot_opt` `term9_list_opt`
+          : `term9`
+   ident_list : `ident_list` `ident`
+              : `ident`
+   appl_arg_list : `appl_arg_list` `appl_arg`
+                 : `appl_arg`
+   appl_arg : ( `ident` := `term` )
+            : `term9`
+   term9 : `term1`
+   term1 : `term0` .( `qualid` `appl_arg_list_opt` )
+         : `term0` .( @ `qualid` `term9_list_opt` )
+         : `term0` % `ident`
+         : `term0`
+   appl_arg_list_opt : `appl_arg_list_opt` `appl_arg`
+                     : `empty`
+   term9_list_opt : `term9_list_opt` `term9`
+                  : `empty`
+   empty :
+   term0 : `qualid` `universe_annot_opt`
+         : `sort`
+         : `numeral`
+         : `string`
+         : _
+         : ? [ `ident` ]
+         : ? [ ?`ident` ]
+         : ?`ident` `evar_instance`
+         : `match_construct`
+         : ( `term` )
+         : {| `record_fields` |}
+         : `{ `term` }
+         : `( `term` )
+         : ltac : ( `ltac_expr` )
+   typeclass_constraint_list_comma : `typeclass_constraint_list_comma` , `typeclass_constraint`
+                                   : `typeclass_constraint`
+   typeclass_constraint : ! `term`
+                        : { `name` } : `exclam_opt` `term`
+                        : `name_colon` `exclam_opt` `term`
+                        : `term`
+   name_colon : `ident` :
+              : _ :
+   exclam_opt : !
+              : `empty`
+   evar_instance : @{ `inst_list_semi` }
+                 : `empty`
+   inst_list_semi : `inst_list_semi` ; `inst`
+                  : `inst`
+   universe_annot_opt : @{ `universe_level_list_opt` }
+                      : `empty`
+   inst : `ident` := `term`
```

Comment:
`inst` is only related to the part I proposed to move to the section on existential variables... *and* it should get a better name, although for this one, I have no suggestion :(